src/Microsoft.Health.Fhir.Shared.Core/Features/Search/SearchOptionsFactory.cs

```diff
@@ -80,6 +80,17 @@ public SearchOptions Create(string resourceType, IReadOnlyList<Tuple<string, str
         {
             var searchOptions = new SearchOptions();
 
+            if (queryParameters != null && queryParameters.Any(_ => _.Item1 == KnownQueryParameterNames.GlobalEndSurrogateId && _.Item2 != null))
```

Comment:
I thought `_` is generally used when the variable isn't used

Reply:
In some cases - yes. I use _ in this type of contexts to clearly indicate that its name does not matter.